apps/www/_blog/2022-08-19-postgrest-v10.mdx

```diff
@@ -0,0 +1,282 @@
+---
+title: 'PostgREST v10: EXPLAIN and Improved Relationship Detection'
+description: "Today, PostgREST 10 was released. Let's take a look at some of the new features that go hand in hand with supabase-js v2."
+author: steve_chavez
+image: lw5/postgrest-v10.png
+thumb: lw5/postgrest-v10.png
+tags:
+  - launch-week
+date: '2022-08-19T07:02:00'
+toc_depth: 3
+---
+
+PostgREST turns your PostgreSQL database automatically into a RESTful API. Today, PostgREST v10 was [released](https://github.com/PostgREST/postgrest/releases/tag/v10.0.0). v10 is not available on the Supabase Platform yet, but it is available for [self-hosting](https://supabase.com/docs/guides/hosting/overview) or as a executable binary from the GitHub release page.
+
+Let's take a look at some of its new features that go hand in hand with [supabase-js v2](https://supabase.com/blog/supabase-js-v2).
+
+## EXPLAIN
+
+Akin to the [PostgreSQL EXPLAIN](https://www.postgresql.org/docs/current/sql-explain.html) command, you can now get the execution plan of a request through REST or `supabase-js`. This works transparently for reads, writes and RPC because every request to PostgREST generates a single SQL query.
```

Comment:
```suggestion
Akin to the [PostgreSQL EXPLAIN](https://www.postgresql.org/docs/current/sql-explain.html) command, you can now get the execution plan of a request through HTTP or `supabase-js`. This works transparently for reads, writes and RPC because every request to PostgREST generates a single SQL query.
```